contracts/0.8.9/Accounting.sol

```diff
@@ -167,22 +146,16 @@ contract Accounting {
     function _simulateOracleReport(
```

Comment:
the comment above is obsolete ⇒ would rather need to state that batches length is zero